app/views/admin/branches/_order.html.erb

```diff
@@ -0,0 +1,31 @@
+<tr class="<%= order_tr_class(order) %>">
+  <td><%= order.id %></td>
+  <td><%= order.user.first_name %></td>
+  <td>
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Name</th>
```

Comment:
No need to show name, quantity text repeatedly

Reply:
Done

Reply:
Done